.eslintrc.cjs

```diff
@@ -1,9 +1,10 @@
-'use strict'
-
 const { overrides } = require('@netlify/eslint-config-node')
 
 module.exports = {
   extends: ['plugin:fp/recommended', '@netlify/eslint-config-node'],
+  parserOptions: {
+    sourceType: 'module',
```

Comment:
The ESLint configuration can be simplified in several ways, now that every package is using pure ES modules.